JitsiConferenceEventManager.js

```diff
@@ -231,6 +231,10 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
     this.chatRoomForwarder.forward(XMPPEvents.RESERVATION_ERROR,
         JitsiConferenceEvents.CONFERENCE_FAILED,
         JitsiConferenceErrors.RESERVATION_ERROR);
+    chatRoom.addListener(XMPPEvents.RESERVATION_ERROR,
+        () => {
+            conference.connection.emuc.doLeave(conference.room.roomjid);
```

Comment:
`conference.connection.emuc` was undefined when I tested this locally. 

Seems to work if I replace this with a rather clumsy looking `conference.connection.xmpp.connection.emuc.doLeave(conference.room.roomjid)`